packages/across-monitor/src/AcrossMonitorConfig.ts

```diff
@@ -0,0 +1,68 @@
+import assert from "assert";
+
+export interface ProcessEnv {
+  [key: string]: string | undefined;
+}
+
+// Set modes to true that you want to enable in the AcrossMonitor bot.
+export interface BotModes {
+  utilizationEnabled: boolean; // Monitors pool utilization ratio
+  unknownRelaysEnabled: boolean; // Monitors relay related events triggered by non-whitelisted addresses
+}
+
+export class AcrossMonitorConfig {
+  readonly bridgeAdminChainId: number;
+
+  readonly pollingDelay: number;
+  readonly errorRetries: number;
+  readonly errorRetriesTimeout: number;
+
+  readonly startingBlock: number | undefined;
+  readonly endingBlock: number | undefined;
+
+  readonly utilizationThreshold: number;
+  readonly whitelistedAddresses: string[];
+  readonly botModes: BotModes;
+
+  constructor(env: ProcessEnv) {
+    const {
+      BRIDGE_ADMIN_CHAIN_ID,
+      POLLING_DELAY,
+      ERROR_RETRIES,
+      ERROR_RETRIES_TIMEOUT,
+      STARTING_BLOCK_NUMBER,
+      ENDING_BLOCK_NUMBER,
+      UTILIZATION_THRESHOLD,
+      WHITELISTED_ADDRESSES,
+      UTILIZATION_ENABLED,
+      UNKNOWN_RELAYS_ENABLED,
+    } = env;
+
+    this.botModes = {
+      utilizationEnabled: UTILIZATION_ENABLED === "true" ? true : false,
+      unknownRelaysEnabled: UNKNOWN_RELAYS_ENABLED === "true" ? true : false,
+    };
+
+    // Default pool utilization threshold at 90%.
+    this.utilizationThreshold = UTILIZATION_THRESHOLD ? Number(UTILIZATION_THRESHOLD) : 90;
+
+    assert(this.utilizationThreshold <= 100, "UTILIZATION_THRESHOLD must be <= 100");
```

Comment:
small nit: can you throw an error rather than asserting? we've found using `assert` produces less predictable errors in prod. see the main relayer [here](https://github.com/UMAprotocol/protocol/blob/011905dcda88889884ec6f377d8b5cea8af20c7e/packages/insured-bridge-relayer/src/RelayerConfig.ts?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L136)

Reply:
fixed